### PR TITLE
Update index_of_rightmost_set_bit.py. Fix grammar mistakes.

### PR DESCRIPTION
**Pull request by Tongcai618**
### Describe your change:

* [ ] Add an algorithm?
* [] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [x] If this pull request resolves one or more open issues then the description above includes the issue number(s) with a [closing keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue): "Fixes #ISSUE-NUMBER".

### CODE REVIEW

**Comment by rohan472000** [comment]
Issue fixed #9078 , now your build will pass.

File: bit_manipulation/index_of_rightmost_set_bit.py [hunk 1]
```diff
@@ -4,8 +4,9 @@
 def get_index_of_rightmost_set_bit(number: int) -> int:
     """
     Take in a positive integer 'number'.
-    Returns the zero-based index of first set bit in that 'number' from right.
-    Returns -1, If no set bit found.
+    Return the zero-based index of the first set bit in this 'number'
+    starting from right.
+    Return -1, if the set bit is not found.
```
Comment (tianyizheng02) [inline bit_manipulation/index_of_rightmost_set_bit.py:9]:
```suggestion
    Return the zero-based index of the first set bit in 'number' starting from right.
    Return -1 if the set bit is not found.
```
Minor formatting

**Review by tianyizheng02 — CHANGES_REQUESTED** [review]

**Comment by rohan472000** [comment]
@Tongcai618 plz run it again to get build pass and apply suggestions provided above, so that it can be merged quickly.

**Comment by Tongcai618** [comment]
> @Tongcai618 plz run it again to get build pass and apply suggestions provided above, so that it can be merged quickly.

**Pull request closed without merging**